Backend/mentii/user_ctrl.py

```diff
@@ -10,6 +10,65 @@
 import hashlib
 from flask import g
 
+def sendForgotPasswordEmail(httpOrigin, jsonData, mailer, dbInstance):
+  email = jsonData.get('email', None)
+  resetPasswordId = str(uuid.uuid4())
+  addResetPasswordIdToUser(email, resetPasswordId, dbInstance)
+  response = ControllerResponse()
+  host = ''
+  if httpOrigin.find('stapp') != -1:
+    host = 'http://stapp.mentii.me'
+  elif httpOrigin.find('app') != -1:
+    host = 'http://app.mentii.me'
+  else:
+    host = 'http://localhost:3000'
+  url = host + '/reset-password/{0}'.format(resetPasswordId)
+  message = render_template('forgotPasswordEmail.html', url=url)
+  #Build Message
+  msg = Message('Mentii: Reset Password', recipients=[email], extra_headers={'Content-Transfer-Encoding': 'quoted-printable'}, html=message)
+  #Send Email
+  mailer.send(msg)
+  return response
+
+def addResetPasswordIdToUser(email, resetPasswordId, dbInstance):
+  table = dbUtils.getTable('users', dbInstance)
```

Comment:
Please add a Boolean return, true for success and false for fail at the end of the function.